tasks/test.yml

```diff
@@ -41,15 +46,24 @@ tasks:
   phpstan:
     dec: Runs PHPStan with mglaman/phpstan-drupal
     cmds:
-      # @todo allow the config to be overridden locally.
-      - ./vendor/bin/phpstan analyse -c vendor/lullabot/drainpipe-dev/config/phpstan.neon {{.PHPSTAN_DIRS}}
+      - ./vendor/bin/phpstan analyse -c vendor/lullabot/drainpipe-dev/config/phpstan.neon {{.TEST_DIRS}}
   phpunit:
     desc: Runs PHPUnit
     cmds:
-      # Ensure directories exist so PHPUnit doesn't complain.
       - |
-        DIRS_ARR=({{.PHPUNIT_DIRS}})
-        for DIR in "${DIRS_ARR[@]}"; do
-          mkdir -p $DIR
-        done
-      - ./vendor/bin/phpunit {{.PHPUNIT_DIRS}}
+        {{ .FUNC_ENSURE_DIRS }}
+      - ./vendor/bin/phpunit {{.TEST_DIRS}}
+  phpcs:
+    desc: Runs PHPCS with Drupal Coding Standards
+    cmds:
+      - |
+        {{ .FUNC_ENSURE_DIRS }}
+      - ./vendor/bin/phpcs --config-set installed_paths vendor/drupal/coder/coder_sniffer
+      - ./vendor/bin/phpcs --standard=vendor/lullabot/drainpipe-dev/config/phpcs.xml {{.TEST_DIRS}}
+  autofix:
```

Comment:
Something I've really appreciated about the Symfony project's approach to coding standards is that they only have standards that can be fixed automatically. This short-circuits much friction on style standards, since you can go so far as fixing them with a bot.

I wonder if there's a way to do something similar with the standards we set by default, to encourage adoption of _something_ by all clients.